src/components/PeriodDimension/utils/RelativePeriodsGenerator.js

```diff
@@ -0,0 +1,140 @@
+import i18n from '@dhis2/d2-i18n'
```

Comment:
Since this file now exports functions instead of a class, by convention it should start with lowercase. Also, maybe it should just be called "relativePeriods"?  (same goes for the FixedPeriodGenerator)

Reply:
Fixed